src/atlas/grid/detail/tiles/FV3Tiles.cc

```diff
@@ -185,17 +185,18 @@ idx_t FV3CubedSphereTiles::tileFromXY( const double xy[] ) const  {
        t = 1;
     } else if ((xy[XX] >= 90.) && ( xy[YY] >= 45.) && (xy[XX] < 180.) && (xy[YY] < 135.)) {
        t = 2;
-    } else if ((xy[XX] >= 180.) && ( xy[YY] > -45.) && (xy[XX] < 270.) && (xy[YY] <= 45.)) {
+    } else if ((xy[XX] >= 180.) && ( xy[YY] >= -45.) && (xy[XX] < 270.) && (xy[YY] <= 45.)) {
```

Comment:
Here, and elsewhere in this block, I needed to add some XX,YY values to some tiles because I was triggering the `ASSERT` at line 204. I probably didn't do this correctly and I'm not sure if this means the `xyt2xy` code is not working correctly.

Reply:
I probably means that xyt2xy is not working properly.  I don't think that you should be changing FV3Tiles.cc.

Reply:
You should not have a point on -45 for cell centers as -45 is on the edge.

Reply:
of the panel

Reply:
These changes to `FV3Tiles` have been removed.